src/phoenix/session/session.py

```diff
@@ -594,6 +620,9 @@ def close_app() -> None:
     _session.end()
     _session = None
     logger.info("Session closed")
+    if reset:
+        logger.info("Resetting working directory")
+        reset_all(hard=True)
```

Comment:
the intent could be stated more explicitly
```suggestion
        delete_all_data(prompt_before_deletion=False)
```

Reply:
changed to `delete_all`

Reply:
probably want to default to set True to prompt before deletion